react/react1/week3/todo-app3/src/components/UserList.js

```diff
@@ -0,0 +1,23 @@
+import React, { useContext } from 'react';
+import { UserContext } from './Context';
+
+function UserList() {
```

Comment:
choose if you're going to use 
function Name( ){ 
 }
 OR
 const Name = ( ) => {
}